packages/select-menu/src/styles.tsx

```diff
@@ -102,4 +84,77 @@ export const TextContainer = styled.div`
   justify-content: center;
   text-align: center;
 `
-export const StyledPopover = styled(Popover)``
+export const InputAsSelectButtonComponent = styled(TextInput)`
+  cursor: pointer;
```

Comment:
Selector control's height should match option's height, also transition between them should be seamless:
![Zrzut ekranu 2019-10-30 o 11 51 01](https://user-images.githubusercontent.com/8265332/67851843-908ecb00-fb0b-11e9-8461-6d7da154c791.png)